### ISSUE TITLE
Triggers panel: play sound on critical alerts

### ISSUE THREAD

**Issue by billbatista** [grafana/grafana-zabbix#255]
Hey.
In Zabbix frontend there's an option to play a sound in specific events, such as critical alerts. I have the new Grafana-Zabbix plugin installed on my Grafana server, and use it to display among other things, the Zabbix Triggers panel on a display.

So I was wondering if it would be possible to customize a sound to play once the panel reads there's a Critical (or other) alert.

**Comment by wmbzz** [comment]
Just realized I should've searched for this. There's already an issue about the same thing, and it's closed. It's not possible to do this, at least for now?

**Comment by alexanderzobnin** [comment]
Hi, @wmbzz 
Torkel is working on Alerting feature in Grafana now. So I'll try to check it after alerting release.

**Comment by wmbzz** [comment]
Hey, @alexanderzobnin, thanks for the quick reply.
Great to now this feature is being developed, cool!

**Comment by kgecici** [comment]
We also need this sound alert feature.

**Comment by alexanderzobnin** [comment]
Alerting branch in Grafana was merged into master. I'll have looked at this feature and will try to integrate it with Zabbix alerts.

**Comment by kkmaslowski** [comment]
@alexanderzobnin any news? It's really needed function for many people. We have TV for displaying triggers from your plugin and other tab from zabbix in background to only play sounds.

**Comment by alexanderzobnin** [comment]
Not sure when I could work on this, but I reopen issue.

**Comment by WilliamCunhaCardoso** [comment]
so, it's not implemented yet?

I need this feature and can't find a way to implement it

**Comment by Zyzto** [comment]
is the feature available now?

**Comment by ttaliesinn** [comment]
Is there any news about this feature?

**Comment by alexanderzobnin** [comment]
Still not a high-priority feature.

**Comment by BrunoBlanes** [comment]
Pleeeeeeeeeeease.

**Comment by RangersMS** [comment]
Any Updates on this? Just to understand the issue, is this project waiting on Grafana to implement an API which allows the zabbix plugin to play sounds or do you want to implement all the sound function into the plugin itself?

**Comment by BrunoBlanes** [comment]
Hi, just to follow up on it, I still keep a Zabbix page open in my local machine just for the alert sound. Is this feature at least in the backlog?
